src/docs/03-Code Guidelines/05-Browser-Support.md

```diff
@@ -1,12 +1,32 @@
 All contents in YouSee DNA is build for and tested in the browsers below. Any YouSee site or web application must be tested in the same browsers.
 
 ## Desktop
-- Internet Explorer 11
-- Chrome (2 latest versions) for Windows and MacOS
-- Firefox (2 latest versions) for Windows and MacOS
-- Safari (2 latest versions) for MacOS
-- Edge (2 latest versions) for Windows
-
-## Mobile/Tablet
-- Chrome (2 latest versions) for iOS and Android
-- Safari (2 latest versions) for iOS
+- Chrome (3 latest versions) for Windows, MacOS and Ubuntu <sup>[[1]](#ref1)</sup>
+- Firefox (3 latest versions) for Windows, MacOS and Ubuntu <sup>[[1]](#ref1)</sup>
+- Safari (2 latest versions) for MacOS <sup>[[2]](#ref2)</sup>
+- Edge (3 latest versions) for Windows <sup>[[1]](#ref1)</sup>
+
+
+## Phone & Tablet
+- Chrome (3 latest versions) for Android and iOS <sup>[[1]](#ref1)</sup>
+- Safari (2 latest versions) for iOS <sup>[[2]](#ref2)</sup>
+- Samsung Internet (latest version) for Android <sup>[[3]](#ref3)</sup>
+
+
+## Partial support
+- Internet Explorer 11 for Windows <sup>[[4]](#ref4)</sup>
+
+
+## No support

```

Comment:
Why have "No support" at all? We've just stated a complete "white list", so there is no need for a "black list" — especially one that contains Opera, which has the same rendering engine as Chrome, so our sites will work just fine in Opera. We just choose not to test in it, as the user base is so small, but that doesn't mean it's "not supported".

Reply:
I like the idea with no support. But I think it should just be
```
Internet Explorer 10
Internet Explorer 9
Internet Explorer 8
```
Just to make it clear that we don't test in these browsers and won't fix bugs.

Reply:
So, keep the section or remove the section? For now I agree with Kim and it's now removed. However, lets discuss it further next time everybody is at the office.